frontend/src/components/JumpBar.js

```diff
@@ -107,6 +107,7 @@ function getPath(type, id) {
     case 'container':  return `/containers/${id}`;
     case 'sample':     return `/samples/${id}`;
     case 'individual': return `/individuals/${id}`;
+    case 'process':    return `/processes/${id}`;
```

Comment:
Just to confirm, protocols/processes don't need to be searchable through the jumpbar?

Reply:
I don't think so ... Users won't have any useful identifier to search for i think.